imbibe.Rproj

```diff
@@ -0,0 +1,22 @@
+Version: 1.0
+
+RestoreWorkspace: No
+SaveWorkspace: No
+AlwaysSaveHistory: Default
+
+EnableCodeIndexing: Yes
+UseSpacesForTab: Yes
+NumSpacesForTab: 2
+Encoding: UTF-8
+
+RnwWeave: knitr
+LaTeX: pdfLaTeX
+
+AutoAppendNewline: Yes
+StripTrailingWhitespace: Yes
+
+BuildType: Package
+PackageUseDevtools: Yes
+PackageInstallArgs: --no-multiarch --with-keep.source
+PackageCheckArgs: --as-cran
+PackageRoxygenize: rd,collate,namespace
```

Comment:
I agree, but I tend to find it helpful to have the Rproj in the GitHub because when I clone then I can just open that file.  The changes to the Rproj are personal though, so I don't know how to not track those areas.

Reply:
OK, let's keep it and I'll try and tweak it to match the code style. That way we're less likely to have lots of whitespace changes appearing as we go back and forth... 😅